docs/source/pkg.rst

```diff
@@ -38,7 +38,7 @@ An example call to install a few packages on 2 hosts (with hostnames like
 
 .. _pkg-remove:
 
-``--remove``
+--remove
```

Comment:
Also, removing the backticks isn't just limited to when there are double-dashes.  I think this looks confusing, for example:

![screen shot 2015-03-19 at 5 26 53 pm](https://cloud.githubusercontent.com/assets/1627176/6741364/428c70b8-ce5d-11e4-85eb-d6d6d4e39908.png)

I just don't think the backticks add anything useful.